### ISSUE TITLE
Request: release a 3.0.11 that allows http_parser 4x

### ISSUE THREAD

**Issue by robbecker-wf** [cfug/dio#1594]
### New Issue Checklist

- [x] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

This isn't a bug so these don't really apply.

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | e.g. flutter / ios / android                          |      |
| Platform Version               | e.g.  1.5.0 / 12.0 / 9.0                              |      |
| Dio Version                    | e.g.  2.1.0 / 1.0.17                                  |      |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

We're having trouble upgrading dependencies in a very large project with many dependencies. It would make these upgrades easier if there were a 3.0.11 version of dio that allowed http_parser 4 in order to be able to resolve to http 0.13.x in our project.
Ultimately we'll need to upgrade to the latest dio 4, but such a release would make it easier for us to get there.

I put together a PR on my own fork to show what that would look like. It's a 1 character change, but since there isn't a 3x branch to target in this repo I couldn't open the PR targeting the main fork. 

https://github.com/robbecker-wf/dio/pull/1

If you're open to this then if you create a 3x branch off of 3.0.10 then I can open a real PR. Or if you have enough info and just want to make the change on your own and publish the version, that'd be great. Thanks for considering this but if you decide you don't want to do this, just respond so that we can take an alternate approach.

**Comment by robbecker-wf** [comment]
ping @wendux  ?

**Comment by robbecker-wf** [comment]
In case you're worried, I have confirmed there are no breaking changes in http_parser 4 
https://github.com/dart-lang/http_parser/compare/

**Comment by robbecker-wf** [comment]
@kuhnroyal ? Anyone?

**Comment by kuhnroyal** [comment]
Nobody can release, please check out https://github.com/cfug/diox

**Comment by robbecker-wf** [comment]
Thanks for the heads up! It did seem really quiet in here.
